Mpumalanga COVID 19 Expenditure Report–Health Donation

```diff
@@ -0,0 +1,145 @@
+DON T-Shirts Large,09-Jul,Goods and services,Imperial Health Sciences,Nkangala District,-
+DON T-Shirts Medium,09-Jul,Goods and services,Imperial Health Sciences,Nkangala District,-
+DON T-Shirts Small,09-Jul,Goods and services,Imperial Health Sciences,Nkangala District,-
+DON T-Shirts X-Large,09-Jul,Goods and services,Imperial Health Sciences,Nkangala District,-
+Double door Fridge to handle the,24-Apr,Kitchen Appliances,Delmas Development Centre(NPO,Bernice Samuel Hospital,48 752.94
+"Equipment Nurses bag, Batteries,",24-Apr,Gaonodd Usn ainfodr mservices,Delmas Development Centre(NPO,Bernice Samuel Hospital,20 000.00
+"Examination gloves, latex, Large",09-Jul,Goods and services,Imperial Health Sciences,Nkangala District,-
+"Examination gloves, latex, Mediu",09-Jul,Goods and services,Imperial Health Sciences,Nkangala District,-
+"Examination gloves, latex, Small",09-Jul,Goods and services,Imperial Health Sciences,Nkangala District,-
+Examinition gloves,19-Jun,Goods and services,AngloAmerican,Impungwe Hospital,-
+Face masks with eye shield,19-Jun,Goods and services,AngloAmerican,Middelburg Pharmaceutical Dep,t                5 000.00
+Face Shields,01-Jul,Goods and services,STEAM POWER S&E AFRICA,Nkangala District,170 496.00
+Gloves Surgical size,19-Jun,Goods and services,AngloAmerican,Middelburg Pharmaceutical Dep,t                          -
+Goggles,19-Jun,Goods and services,AngloAmerican,Middelburg Pharmaceutical Dep,t                   500.00
+Googles,09-Jul,Goods and services,Imperial Health Sciences,Nkangala District,-
+Gowns / Disposable,19-Jun,Goods and services,AngloAmerican,Middelburg Pharmaceutical Dep,t                1 000.00
+Hand sanitiser Spray,19-Jun,Goods and services,AngloAmerican,Middelburg Pharmaceutical Dep,t                1 000.00
+Hand Paper towels,24-Apr,Goods and services,Delmas Development Centre(NPO,Bernice Samuel Hospital,-
+HP Laser jet Multifunction 137FW,01-Jul,rPor i7n2t2e0r,South32 SA Coal Holdings Propriet,rNyk Laimngiatelad District,4 232.54
+Industrial Paper Towels,15-May,Goods and services,Glencore Operations S.A (Pty) Ltd,Nkangala District,37 500.00
+Internet router bundle for 06 mon,01-Jul,Goods and services,South32 SA Coal Holdings Propriet,rNyk Laimngiatelad District,5 085.30
+Sanitisers,(blank),Goods and services,STEAM POWER S&E AFRICA,Nkangala District,79 500.00
+KN95/ FFP2 Masks,10-Jun,Goods and services,FORD,Nkangala District,-
+KN95/ FFP2 Masks,09-Jul,Goods and services,Naspers,Nkangala District,-
+7 x Logitech Wireless USB Combo,01-Jul,aCrodm &p Mutoeur sDee-1sk2t8op,South32 SA Coal Holdings Propriet,rNyk Laimngiatelad District,3 566.78
+7 x Macaroni Versitas Lightweight,01-Jul,Gposoed sB acnkdp saecrkvices,South32 SA Coal Holdings Propriet,rNyk Laimngiatelad District,1 522.78
```

Comment:
Spot checks showed some issues with the above 2 rows:
`aCrodm &p Mutoeur sDee-1sk2t8op` should be `Computer Desktop`
`Gposoed sB acnkdp saecrkvices` should be `Goods and Services`